deps/spidershim/src/v8trycatch.cc

```diff
@@ -41,7 +41,7 @@ struct TryCatch::Impl {
     // TODO: Propagate exceptions to the caller
     //       https://github.com/mozilla/spidernode/issues/51
   }
-  Isolate* Isolate() const { return isolate_; }
+  Isolate* Iso() const { return isolate_; }
```

Comment:
On Thu, May 05, 2016 at 11:42:54AM -0700, Ehsan Akhgari wrote:
> In deps/spidershim/src/v8trycatch.cc:
> 
> > @@ -41,7 +41,7 @@ struct TryCatch::Impl {
> >      // TODO: Propagate exceptions to the caller
> >      //       https://github.com/mozilla/spidernode/issues/51
> >    }
> > -  Isolate* Isolate() const { return isolate_; }
> > +  Isolate* Iso() const { return isolate_; }
> 
> OK, I'll fix this later myself. What version of gcc did you get this error
> with?

I think everyone on my machine so 4.9 / 5 / 6.  At least with a reduced
test case I used to figure out what was going on.